Include/Rover/Autobox.hpp

```diff
@@ -1,5 +1,7 @@
 #ifndef ROVER_AUTOBOX_HPP
 #define ROVER_AUTOBOX_HPP
+#include "Box.hpp"
```

Comment:
Includes must be of the form ```#include "Rover/xxx``` so that this library will work when it's used in other applications.